clouddriver-openstack/src/test/groovy/com/netflix/spinnaker/clouddriver/openstack/cache/KeysSpec.groovy

```diff
@@ -16,11 +16,12 @@
 
 package com.netflix.spinnaker.clouddriver.openstack.cache
 
-import com.netflix.spinnaker.clouddriver.core.provider.agent.Namespace
-import com.netflix.spinnaker.clouddriver.openstack.OpenstackCloudProvider
 import spock.lang.Specification
 import spock.lang.Unroll
 
+import static com.netflix.spinnaker.clouddriver.openstack.OpenstackCloudProvider.ID
+import static com.netflix.spinnaker.clouddriver.openstack.cache.Keys.Namespace.*
```

Comment:
Can you resolve the * import? Not sure what the standard is in spinnaker, but generally, it is not preferred to use * imports